src/status_im2/contexts/onboarding/create_password/view.cljs

```diff
@@ -165,7 +165,8 @@
          [rn/view {:style style/bottom-part}
           [rn/view {:style style/disclaimer-container}
            [quo/disclaimer
-            {:on-change #(reset! accepts-disclaimer? %)
+            {:blur?     true
```

Comment:
Not related to this PR, this is the disclaimer showed in the password creation screen, this propery (`blur?`) had been accidentally erased  recently